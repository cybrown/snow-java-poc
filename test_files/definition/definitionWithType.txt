def a: int = 3