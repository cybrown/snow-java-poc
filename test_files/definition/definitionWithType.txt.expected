<Definition>
    <Identifier>a</Identifier>
    <Identifier>int</Identifier>
    <LiteralInteger>3</LiteralInteger>
</Definition>
